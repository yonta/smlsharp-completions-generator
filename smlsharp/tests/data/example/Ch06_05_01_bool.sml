1 < 2;
1 < 2 andalso 1 > 2;
1 < 2 orelse 1 > 2;

if 1 < 2 then 1 else 2;

(if 1 < 2 then 1 else 2) * 10;
