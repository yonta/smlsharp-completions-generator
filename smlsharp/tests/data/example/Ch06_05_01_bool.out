val it = true : bool

val it = false : bool

val it = true : bool

val it = 1 : int

val it = 10 : int
